src/ResourceManager/Relay/Commands.Relay/Cmdlets/CheckNameAvailability/TestAzureRelayCheckNameAvailability.cs

```diff
@@ -0,0 +1,43 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
```<nit>```For .Net coding Guildines compatibility, you should change the name fo the file to match the name of the class ```</nit>```